src/shared/bg_misc.cpp

```diff
@@ -459,33 +459,6 @@ evolveInfo_t BG_ClassEvolveInfoFromTo( const int from, const int to )
 	return { classIsUnlocked, isDevolving, evolveCost };
 }
 
-/*
```

Comment:
Can you add context in your commit messages for why we are making these changes

Reply:
I've added some text in the commit message. I thought the comment on the function it removes would be suggestive enough `// [this] function will always return true because it will notice you can devolve to dretch or granger, even when far from the overmind`

Reply:
Yes, but its better to have it in the commit message :D